plugins/SitesManager/lang/en.json

```diff
@@ -191,6 +191,17 @@
         "SiteWithoutDataStartTrackingDataDescriptionLine1": "Choose your preferred tracking method below. If you need help, you can share these %1$sinstructions via email%2$s or %3$sinvite another user%4$s for assistance.",
         "SiteWithoutDataStartTrackingDataDescriptionLine2": "Note: This screen will disappear as soon as some data is tracked for this website. Alternatively, you can %1$sclick here to hide this page for one hour%2$s.",
         "EmailInstructionsButtonText": "Email instructions",
-        "SiteWithoutDataIgnorePage": "Don't show this page for an hour"
+        "SiteWithoutDataIgnorePage": "Don't show this page for an hour",
+        "SiteWithoutDataVue": "Vue.js",
+        "SiteWithoutDataVueDescription": "You can use the \"%1$s\" npm package to integrate Matomo to track data, follow the instructions from this %2$sguide%3$s.",
+        "VueDetected": "Vue.js was detected on your website. Did you know you can use the \"%1$s\" npm package to integrate Matomo into your site? %2$sLearn more in this guide%3$s.",
+        "SiteWithoutDataVueIntro": "Tracking your Vue.js website or app is easy with Matomo, even for Single Page Applications.",
+        "SiteWithoutDataVueFollowStep1": "Install the \"%1$s\" npm package.",
+        "SiteWithoutDataVueFollowStep2": "Initialize %1$s in %2$s with your Matomo Instance details (example code below).",
+        "SiteWithoutDataVueFollowStep2ExampleCode": "Example code for %1$s",
+        "SiteWithoutDataVueFollowStepCompleted": "%1$sCongratulations!%2$s You have successfully installed the Matomo Analytics tracking code via the \"%3$s\" npm package. To verify that hits are being tracked, visit your website and check that this data is visible in your Matomo instance.",
+        "SiteWithoutDataVueFollowStepNote1": " Now you can access Matomo in your components through %1$s.",
+        "SiteWithoutDataVueFollowStepNote2": "If you followed these steps and have no data coming into Matomo, you can contact our support team for assistance or try the %1$sMatomo Tag Manager%2$s instead (use this link for %3$sSingle Page Applications%4$s).",
```

Comment:
Is it on purpose to advise on premise users to contact support?

Reply:
@sgiehl the copy was given by PO, but doesn't make sense to mention support here, will override this translation in Cloud plugin.